paimon-core/src/main/java/org/apache/paimon/operation/FileStoreCommit.java

```diff
@@ -72,4 +73,7 @@ void overwrite(
      * @param partitions A list of partition {@link Map}s. NOTE: cannot be empty!
      */
     void dropPartitions(List<Map<String, String>> partitions, long commitIdentifier);
+
+    /** Abort an unsuccessful commit. The data files will be deleted. */
+    void abort(List<CommitMessage> commitMessages);
```

Comment:
Could the param use `ManifestCommittable committable`?

Reply:
`ManifestCommittable` need an identifier. But here we don't need that.